docs/content/en/docs/getting-started/optional/kubeletconfig.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "KubeletConfiguration"
+linkTitle: "KubeletConfiguration"
+weight: 40
+aliases:
+    /docs/reference/clusterspec/optional/kubeletconfig/
+description: >
+  EKS Anywhere cluster yaml specification for Kubelet Configuration
+---
+
+## Kubelet Configuration Support
+
+#### Provider support details
+|                     | vSphere | Bare Metal | Nutanix | CloudStack | Snow |
+|:-------------------:|:-------:|:----------:|:-------:|:----------:|:----:|
+|    Ubuntu 20.04     |    ✔    |     ✔      |    ✔    |     —      |  —   |
+|    Ubuntu 22.04     |    ✔    |     ✔      |    ✔    |     —      |  —   |
+| Bottlerocket        |    ✔    |     —      |    —    |     —      |  —   |
+|      RHEL 8.x       |    ✔    |     ✔      |    ✔    |     ✔      |  —   |
+|      RHEL 9.x       |    —    |     —      |    ✔    |     ✔      |  —   |
+
+
+You can configure EKS Anywhere to specify Kubelet settings and configure those for control plane and/or worker nodes starting from `v0.20.0`. This can be done using `kubeletConfiguration`.
```

Comment:
We now have versioned docs, so do we need to explicitly mention the version here?

Reply:
Yeah I couldn't find the specific v0.20.0 docs on our docs page so I added this extra info for now, can probably remove it later on.